Makefile

```diff
@@ -141,14 +141,24 @@ batcher_send_groth16_task: batcher/client/target/release/batcher-client
 		--public_input ../../test_files/groth16/ineq_1_groth16.pub \
 		--vk ../../test_files/groth16/ineq_1_groth16.vk \
 
-batcher_send_infinite_groth16: ./batcher/client/target/release/batcher-client ## Send a different Groth16 BN254 proof using the task sender every 3 seconds
-	@mkdir -p task_sender/test_examples/gnark_groth16_bn254_infinite_script/infinite_proofs
+batcher_send_plonk_task: batcher/client/target/release/batcher-client
+	@echo "Sending PlonkBn254 1!=0 task to Batcher..."
+	@cd batcher/client/target/release && ./batcher-client \
+		--proving_system GnarkPlonkBn254 \
+		--proof ../../test_files/groth16/ineq_1_groth16.proof \
+		--public_input ../../test_files/plonk_bn254/ineq_1_plonk_bn254.pub \
+		--vk ../../test_files/groth16/ineq_1_plonk_bn254.vk \
+
+batcher_send_infinite_groth16_tasks: ./batcher/client/target/release/batcher-client ## Send a different Groth16 BN254 proof using the task sender every 3 seconds
+	@mkdir -p task_sender/test_examples/gnark_groth16_bn254_infinite_script/infinite_proofs	
```

Comment:
call using cargo run, as in [this PR](https://github.com/yetanotherco/aligned_layer/pull/218).

This makes rust to compile if necesarry